.releaserc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "branches": [{ "name": "main" }, { "name": "alpha", "prerelease": true }],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/changelog",
+    [
+      "@semantic-release/npm",
+      {
+        "tarballDir": "release"
+      }
+    ],
+    [
+      "@semantic-release/github",
+      {
+        "assets": "release/*.tgz"
+      }
+    ],
+    "@semantic-release/git"
+  ]
+}
```

Comment:
Since we have a `.releaserc.json` file, the `release.config.js` file can be removed.